amazonfreertossdk/src/main/java/software/amazon/freertos/amazonfreertossdk/AmazonFreeRTOSConstants.java

```diff
@@ -78,7 +78,10 @@
     }
 
     public enum AmazonFreeRTOSError {
-        BLE_DISCONNECTED_ERROR
+        BLE_NO_ERROR,
+        BLE_DISCONNECTED_ERROR,
+        BLE_EMPTY_BROKER_ENDPOINT,
+        BLE_EMPTY_MQTT_CLIENT_ID
```

Comment:
Could we expand on more error codes like cognito authentication error ?

Reply:
Cognito authentation failure was initially ommitted because it required some more involved error handoffs

However, I added ability to report cognito auth failure in latest commit.